community/caddy/APKBUILD

```diff
@@ -29,22 +29,18 @@ prepare() {
 }
 
 build() {
-	cd "$builddir"
 	export GOPATH="$srcdir"
 	go build -v -o bin/caddy -ldflags \
 		"-X github.com/mholt/caddy/caddy/caddymain.gitTag=$pkgver" \
 		./caddy
 }
 
 check() {
-	cd "$builddir"
```

Comment:
please keep it to minify changes on backports

Reply:
Reverted.